modules/lib/src/main/resources/assets/styles/project/project-access-control.less

```diff
@@ -39,6 +39,7 @@
       float: left;
       margin: 0 10px 0 0;
       min-width: 200px;
+      width: 70%;
```

Comment:
I would try to avoid using width for it, and update `.access-selector` to have a `flex-shrink: 0`